llama_index/logger/__init__.py

```diff
@@ -1,5 +1,40 @@
-"""Init params."""
+"""
```

Comment:
Unified `logger` is defined here. This seems like the best place for documentation to exist?